doc/community-build/Building-CORTX-From-Source-for-SingleNode.md

```diff
@@ -11,7 +11,7 @@ The CORTX deployment and configuration is a four-step procedure:
 
 ## Prerequisites
 
--   Create a [CentOS 7.8.2003](http://repos-va.psychz.net/centos/7.8.2003/isos/x86_64/) VM with following minimum configuration:
+-   Create a [CentOS 7.9.2009](http://repos-va.psychz.net/centos/7.9.2009/isos/x86_64/) VM with following minimum configuration:
```

Comment:
We need to support both CentOS 7.8.2003 and CentOS 7.9.2009.  We also need to modify it at other places. 
e.g. 
In https://github.com/Seagate/cortx/blob/main/doc/community-build/Generate-Cortx-Build-Stack.md#procedure We need to use docker image based on VM OS.

Reply:
PR# 1170 updated, will raise another PR for other link to get it fixed